gptel.el

```diff
@@ -1118,6 +1131,32 @@ Valid JSON unless NO-JSON is t."
 
 ;;; Saving and restoring state
 
+(defun gptel--restore-props (bounds-alist)
+  "Restore text properties from BOUNDS-ALIST.
```

Comment:
This description is confusing even though I understand the alist structure.  Please rewrite it, or add a note to rewrite it later and I'll get to it.

Reply:
22af547